README.md

```diff
@@ -1 +1,181 @@
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- <a href="https://nodejs.org/en/download">Node.js</a>
+- <a href="https://docs.npmjs.com/downloading-and-installing-node-js-and-npm">npm</a>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd desired-folder
+  git clone https://github.com/henokkhm/microverse-bookstore-app.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd microverse-bookstore-app
+  npm install
+```
+
+### Usage
+
+- To start the app in the development mode on your local machine, simply run:
+
+`npm start`
+
+Then open [http://localhost:3000](http://localhost:3000) to view it in your browser. The page will reload when you make changes. You may also see any lint errors in the console.
+
+- To run the linters, run: 
+
+`npm run lint`
+
+### Run tests
+
+
+- To run the tests, run the following script from the root of the project: 
+  
+`npm test`
+
+This will launch the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+
+### Deployment
+
+To build static assets for deployment, run the following script from the root of the project:
+
+`npm run build`
+
+The static assets will be saved in the `build` folder. The build is minified and the filenames include the hashes. Your app is ready to be deployed!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Henok K Hailemariam**
+
+- GitHub: [@henokkhm](https://github.com/henokkhm)
+- Twitter: [@henokkhm](https://twitter.com/henokkhm)
+- LinkedIn: [henokkhm](https://www.linkedin.com/in/henokkhm/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add more content and styling**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, you can contribute to it. You are also welcome to give it a star on GitHub.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for teaching and enforcing professional level practices.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] You haven't added the `LICENCE.md` file that's why it is showing errors. Kindly fix this issue to make your readme more professional.

![image](https://github.com/henokkhm/microverse-bookstore-app/assets/60083437/53a9ac17-0762-4e11-b3fd-4b5637e161a1)